Intersect.Client/Entities/Player.cs

```diff
@@ -2094,8 +2212,8 @@ private void ProcessDirectionalInput()
                     {
                         if (MoveDir != Dir)
                         {
-                            Dir = (byte)MoveDir;
-                            PacketSender.SendDirection(Dir);
+                            Dir = MoveDir;
+                            PacketSender.SendDirection((byte)Dir);
```

Comment:
Can you just change SendDirection to accept `Direction` (and it can do the casting to byte if needed inside of there)